### PR TITLE
Make the run script work with multiple JVM installations

### PR DESCRIPTION
**Pull request by mkoncek**

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -1,7 +1,14 @@
 #!/bin/sh
-set -eu
+set -e
```
Comment (mizdebsk) [inline run.sh:2]: You can keep `-u` and evaluate variables that can be possible unset like this: `${JAVA_HOME:-}`

**Review by mizdebsk — CHANGES_REQUESTED** [review]
I think that the PR goes in a wrong direction. The `run.sh` script is used for local testing only, after doing Maven build. It even refers to the `target` directory created by Maven. Appropriate JAVA_HOME needs to be set for Maven anyway, so why not use the same JVM in run.sh? I will provide example of what I mean.

**Comment by mizdebsk** [comment]
> I will provide example of what I mean.

See #167 for example of what I mean as counter-proposal.

**Comment by mkoncek** [comment]
Closing in favor of your solution.

**Pull request closed without merging**